src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1656,6 +1657,24 @@ private static void ValidateWin32Settings(string win32ResourceFile, string win32
             }
         }
 
+        internal static IEnumerable<InstrumentationKind> ParseInstrumentationKinds(string value, IList<Diagnostic> diagnostics)
```

Comment:
No need to change from iterator. It looks like that is the pattern for `Parse` methods.